lib/active_merchant/billing/gateways/wompi.rb

```diff
@@ -141,6 +142,10 @@ def add_basic_card_info(post, card, options)
         post[:cvc] = cvc if cvc && !cvc.empty?
       end
 
+      def add_tip_in_cents(post, options)
+        post[:tip_in_cents] = options[:tip_in_cents] if options[:tip_in_cents]
```

Comment:
I notice that the other places where `amount_in_cents` is being set, there's a `to_i` being called to ensure that we're passing an integer. I wonder if we need to do the same here for `tip_in_cents`, in the event that an amount is passed in as a string?

Reply:
I added the `to_i` just in case. Tested it by changing amount in remote test to a String and it behaved as expected. Did not keep that change permanent or add a separate test because it seemed a bit overkill but can do that if you think it would be useful.